pkg/serverless/tags/tags_test.go

```diff
@@ -212,37 +212,39 @@ func TestAddTagInvalid(t *testing.T) {
 	assert.Equal(t, "value_b", tagMap["key_b"])
 }
 
-func TestAddTagInvalid2(t *testing.T) {
+func TestAddTagInvalidEmpty(t *testing.T) {
 	tagMap := map[string]string{
 		"key_a": "value_a",
 		"key_b": "value_b",
 	}
-	addTag(tagMap, "invalidTag:invalid:invalid")
```

Comment:
@maxday in this test it seems that tags with column are not considered valid on purpose. This is a different behavior from https://github.com/DataDog/dd-trace-js/blob/39656348312aee8eb62be8fa260212772bf6e997/packages/dd-trace/test/config.spec.js#L465

is `serverless` going to act differently from `serverless-init` or `tracer` on purpose regarding this type of tags?

Reply:
cc @vickenty